contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -89,6 +88,26 @@ void multisig::approve( name proposer, name proposal_name, permission_level leve
             a.requested_approvals.erase( itr );
          });
    }
+
+   transaction_header trx_header = get_trx_header(prop.packed_transaction.data(), prop.packed_transaction.size());
+
+   if ( prop.earliest_exec_time.has_value() ) {
+      if ( prop.earliest_exec_time.value().has_value() ) {
+         return;
+      } else {
+         auto table_op = [](auto&&, auto&&){};
+         if ( approvals_satisfy_trx_authorization(proposer, proposal_name, prop.packed_transaction, table_op) ) {
+            auto prop_it = proptable.find( proposal_name.value );
+            proptable.modify( prop_it, proposer, [&]( auto& p ) {
```

Comment:
There is no need to call `proptable.find` here. You can just pass in the `prop` you retrieved earlier in this function rather than `prop_it`.

Reply:
Previously linked: https://github.com/EOSIO/eosio.contracts/blob/deprecate-deferred3/contracts/eosio.msig/src/eosio.msig.cpp#L107-L116